src/classes/traits/proteus.php

```diff
@@ -131,41 +135,49 @@ public function purchase_link( $utm_medium = '' ) {
 		public function banner() {
 			$days = $this->date_diff();
 
+			// Default header and text
+			$header = sprintf( 'Your sandbox for WoonderShop is expiring in %1$s days!', $days );
+			$text   = esc_html__( 'Once you\'re finished with testing, you can export the changes and import them to a store on your own hosting. Or, you can continue with WooCart by subscribing to the 7-day free trial below. WoonderShop theme is included for free!', 'woocart-defaults' );
+
 			$buttons = array(
 				0 => array(
-					esc_html__( 'Learn more about WooCart »', 'woocart-defaults' ),
-					'https://woocart.com/pricing?plan=trial&store_id=' . $_SERVER['STORE_ID'],
-				),
-				1 => array(
 					esc_html__( 'Buy the theme for $79', 'woocart-defaults' ),
 					$this->purchase_link(),
 				),
+				1 => array(
+					esc_html__( 'Keep hosting on WooCart', 'woocart-defaults' ),
+					'https://woocart.com/pricing?plan=trial&store_id=' . $_SERVER['STORE_ID'],
```

Comment:
@dmurko any update on Paddle passthrough with affiliate?

Reply:
Paddle confirmed aff link cannot pass it and they did not see an obvious solution. What if we send to our LP, cookie the storeID, then send them through the aff link?

https://niteo.slack.com/archives/C85P09277/p1574949124007600

Reply:
Injecting affiliate information via JS is not possible and there is no way to generate affiliate links programmatically. So, I think that's how we can get `store_id` and track affiliate commission.

Reply:
```suggestion
					'https://woocart.com/paddle/proteus?store_id=1234?store_id=' . $_SERVER['STORE_ID'],
```

Reply:
Change made.